app/controllers/spree/admin/payments_controller.rb

```diff
@@ -153,7 +153,7 @@ def load_payment
       def authorize_stripe_sca_payment
         return unless @payment.payment_method.class == Spree::Gateway::StripeSCA
 
-        @payment.authorize!(spree.order_path(@payment.order, only_path: false))
+        @payment.authorize!(main_app.order_path(@payment.order, only_path: false))
```

Comment:
Do the specs still pass if `main_app` is removed here?

Reply:
I bet this is because they are within the spree engine namespace.